ravenpy/models/emulators.py

```diff
@@ -630,9 +650,11 @@ def ost2raven(self, ops):
         BLENDEDParams named tuple
           Parameters expected by Raven.
         """
-        names = ["par_x{:02}".format(i) for i in range(1, 36)]+["par_r{:02}".format(i) for i in range(1, 9)]
-        names[3]  = "pow_x04"
-        names[9]  = "sum_x09_x10"
+        names = ["par_x{:02}".format(i) for i in range(1, 36)] + [
+            "par_r{:02}".format(i) for i in range(1, 9)
```

Comment:
You can use an f-string instead of `format` here: `f"par_x{i:02}"`

Reply:
Sure but then we should change this in ALL other tests too.